etc/behat/services/contexts/setup.xml

```diff
@@ -22,13 +22,15 @@
         <parameter key="sylius.behat.context.setup.product.class">Sylius\Behat\Context\Setup\ProductContext</parameter>
         <parameter key="sylius.behat.context.setup.product_review.class">Sylius\Behat\Context\Setup\ProductReviewContext</parameter>
         <parameter key="sylius.behat.context.setup.promotion.class">Sylius\Behat\Context\Setup\PromotionContext</parameter>
+        <parameter key="sylius.behat.context.setup.province.class">Sylius\Behat\Context\Setup\ProvinceContext</parameter>
         <parameter key="sylius.behat.context.setup.security.class">Sylius\Behat\Context\Setup\SecurityContext</parameter>
         <parameter key="sylius.behat.context.setup.shipping.class">Sylius\Behat\Context\Setup\ShippingContext</parameter>
         <parameter key="sylius.behat.context.setup.taxation.class">Sylius\Behat\Context\Setup\TaxationContext</parameter>
         <parameter key="sylius.behat.context.setup.taxonomy.class">Sylius\Behat\Context\Setup\TaxonomyContext</parameter>
         <parameter key="sylius.behat.context.setup.theme.class">Sylius\Behat\Context\Setup\ThemeContext</parameter>
         <parameter key="sylius.behat.context.setup.user.class">Sylius\Behat\Context\Setup\UserContext</parameter>
         <parameter key="sylius.behat.context.setup.zone.class">Sylius\Behat\Context\Setup\ZoneContext</parameter>
+        <parameter key="sylius.behat.context.setup.zone_member.class">Sylius\Behat\Context\Setup\ZoneMemberContext</parameter>
```

Comment:
I have not seen it (the class) yet, but ZoneMember always belongs to a zone and I can't see why we would need a separate context for that.